index.js

```diff
@@ -24,15 +24,28 @@ function browserSyncPlugin(options){
             }
         });
 
-        //Require options that are non-negotiable
-        bsOptions.middleware = function (req, res, next) {
-            metalsmith.build(function (err) {
+        var rebuild = function() {
+            if (bs.paused) {
```

Comment:
I prefer to have the reverse case instead. I'm not really a fan of empty returns for the sake of argument checking. Could you update the logical branch to be reversed (e.g. !bs.paused)?